subcommands/targets/offline-update.go

```diff
@@ -100,6 +111,21 @@ and is printed by this command.`,
 	parentCmd.AddCommand(signCmd)
 }
 
+func initCheckCmd(parentCmd *cobra.Command) {
+	checkCmd := &cobra.Command{
+		Use:   "check <path to an offline bundle>",
+		Short: "Check and print info about an offline bundle",
+		Long: `Check and print info about an offline bundle.
```

Comment:
Looking below this command seems to be more _informative_ than _verifying_.

Usually, we use the `show` sub-command for this kind of an operation.
In one specific use case I've selected a `review` sub-command name for a very complex use case of TUF root updates.

So, instead of introducing one more verb - `check`, I suggest to select between these two:
- `show` if you think that it is as regular as e.g. `waves show` or `keys ca show`.
- `review` if you think that it is as special as `keys tuf updates review`.

Reply:
The verb `check` does not necessary and only mean "to verify" something, it can be used for the "to find out" meaning too.

I suppose, for the sake of consistency with the rest of the code base I think it makes sense to rename it to `show`.

Reply:
Fixed.